src/tensorlake/function_executor/user_events.py

```diff
@@ -51,6 +49,49 @@ def log_user_event_initialization_finished(
     )
 
 
+def log_user_event_initialization_failed(
+    details: InitializationEventDetails, error: BaseException
+) -> None:
+    # Using standardized tags, see https://github.com/tensorlakeai/indexify/blob/main/docs/tags.md.
+    print_cloud_event(
+        {
+            "level": "error",
+            "event": "function_executor_initialization_failed",
+            "message": "Function executor initialization failed",
+            "namespace": details.namespace,
+            "application": details.application_name,
+            "application_version": details.application_version,
+            "function": details.function_name,
+            "error": clean_stack_trace(error),
+        }
+    )
+
+
+def clean_stack_trace(error: BaseException) -> list[str]:
```

Comment:
I am guessing it wasn't possible to surround the code which loads the user code with a try-catch?

Reply:
we already have a try catch on that, but it's surrounded with other stack trace information from python's standard library. We cannot do a try/catch inside the stdlib.